.gitignore

```diff
@@ -1,5 +1,6 @@
 .DS_Store
 .vscode/
+.nova/
```

Comment:
What's this?

Reply:
oh editor configs: https://panic.com/nova/